### PR TITLE
WT-7075 Prevent MongoDB yielding in the LSWA workload

### PR DESCRIPTION
**Pull request by tetsuo-cpp**
@sulabhM 
I'm just putting this here to make it visible to you.

We're not seeing much lookaside usage yet but the transaction length looks good and the yielding has been solved. Reducing the document count and increasing the size hasn't helped as we'd anticipated.

### CODE REVIEW

File: src/cast_core/src/CollectionScanner.cpp
Comment (tetsuo-cpp) [inline src/cast_core/src/CollectionScanner.cpp:110]: Necessary to prevent yielding on the `getMore`.

File: src/phases/scale/LargeScalePhases.yml
Comment (tetsuo-cpp) [inline src/phases/scale/LargeScalePhases.yml:17]: For debugging.

File: src/phases/scale/LargeScalePhases.yml
Comment (tetsuo-cpp) [inline src/phases/scale/LargeScalePhases.yml:68]: Creating larger docs.

File: src/phases/scale/LargeScalePhases.yml
Comment (tetsuo-cpp) [inline src/phases/scale/LargeScalePhases.yml:242]: I wanted to increase the scan rate (which necessitates increasing the size if we want the same duration).

**Comment by tetsuo-cpp** [comment]
I'm running with this script to prevent yielding:

```
#!/bin/bash

killall -9 mongod; rm -rf rplset/rs0/mongod.log; rm -rf rplset/rs0/db/; mkdir rplset/rs0/db; bin/mongod -f mongod.conf; sleep 1; bin/mongo --eval "rs.initiate()"; bin/mongo --eval "db.adminCommand({setParameter: 1, internalQueryExecYieldIterations: 2147483647})"; bin/mongo --eval "db.adminCommand({setParameter: 1, internalQueryExecYieldPeriodMS: 2147483647})"; sleep 2
```

File: src/cast_core/src/CollectionScanner.cpp
Comment (rtimmons) [reply src/cast_core/src/CollectionScanner.cpp:110]: Consider adding this comment to the source for the next maintainer 😄

**Review by rtimmons — APPROVED** [review]
LGTM assuming a successful patch-build.

(Hopefully you weren't waiting on me to merge - sorry if you were. In general getting LGTM from STM on individual actors/workload yamls isn't necessary, but we're happy to be another set of eyes if there's any uncertainty.)

**Comment by tetsuo-cpp** [comment]
> (Hopefully you weren't waiting on me to merge - sorry if you were. In general getting LGTM from STM on individual actors/workload yamls isn't necessary, but we're happy to be another set of eyes if there's any uncertainty.)

No worries! We're in the middle of debugging the workload's interaction with Mongo so that's why it's been sitting here.

**Comment by tetsuo-cpp** [comment]
@rtimmons The failed job is also failing in Waterfall so I'm going to go ahead and merge this.

**Pull request merged**
